Empire/AdminSettings.php

```diff
@@ -129,6 +131,10 @@ public function showSettings() {
                 <p><label>% of ads on Empire: <input type="text" name="empire_percent" id="empire_percent" value="<?php echo $empire_test; ?>" /></label></p>
                 <p><label>Key-Value for Split Test: <input type="text" name="empire_value" id="empire_value" value="<?php echo $empire_value; ?>" /></label></p>
 
+                <p><label><input type="checkbox" name="empire_amp_ads_enabled"
+                                 id="empire_amp_ads_enabled" <?php echo $amp_ads_enabled ? 'checked' : ''; ?>> AMP Ads
+                        Enabled</label></p>
```

Comment:
The tabbing here should probably be cleaned up 👍  If it's a line length issue on the `<input />` itself, you could always put the attributes on new lines like we do with JSX:

```php
<input
  type="checkbox"
  name="empire_amp_ads_enabled"
  id="empire_amp_ads_enabled"
  <?php echo $amp_ads_enabled ? 'checked' : ''; ?>
>
```

Also, this input is missing the trailing `/>`; we always include them if this is self-closing